### ISSUE TITLE
Multiple `use` classes with CDI

### ISSUE THREAD

**Issue by caarlos0** [mapstruct/mapstruct#930]
We have a mapper like this:

``` java
@Mapper(componentModel = "cdi", uses = { FooMapper.class, BarMapper.class, BlahMapper.class })
public interface FooMapper {
```

But the generated code is injecting only the last mapper defined in `uses`:

``` java
@Generated(
    value = "org.mapstruct.ap.MappingProcessor",
    date = "2016-10-17T11:25:30-0200",
    comments = "version: 1.0.0.Final, compiler: javac, environment: Java 1.7.0_80 (Oracle Corporation)"
)
@ApplicationScoped
public class FooMapperImpl implements FooMapper {

    @Inject
    private BlahMapper blahMapper;

    @Override
```

We are using MapStruct 1.0.0.Final and java 7.

**Comment by gunnarmorling** [comment]
Hum, that seems like a bug. Could you check 1.1.0.CR1 and let us know
whether you see the issue there, too?

**Comment by caarlos0** [comment]
@gunnarmorling it broke some other stuff, I can't try to fix them right now to reproduce the issue...

**Comment by caarlos0** [comment]
I'll try to do a unit test for that

**Comment by gunnarmorling** [comment]
Hey @caarlos0, any luck with creating a reproducer for this one?

**Comment by caarlos0** [comment]
hi @gunnarmorling , I didn't get to start it yet... feel free to go ahead if you have some free time

**Comment by filiphr** [comment]
I think that this one might be the same as #938, or most probably it has already been fixed in the `1.1.0.Final`

**Comment by filiphr** [comment]
Closing due to lack of requested feedback. If you provide us with more information we would be more than happy to reopen the issue
